xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -26,6 +26,8 @@
 #include "guilib/guiinfo/GUIInfoLabels.h"
 #include "powermanagement/PowerManager.h"
 #include "profiles/ProfileManager.h"
+#include "pvr/PVRManager.h" // Needed to get the PVR client count
+#include "pvr/addons/PVRClients.h" // Needed to get the PVR client count
```

Comment:
Remove the comments here, please One can easily look up in the code for what the includes are needed and those comments always tend to get outdated in the futire, because nobody remembers to update the comment if something changes wrt pvr usage in this source file.

Reply:
Done.